### PR TITLE
Starting the influxdb container in detached mode. Adding flag

### PR DESCRIPTION
**Pull request by VijetaH**
Closes #

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by VijetaH** [comment]
@sanderson Makes sense if this is by choice. I removed the -d flag from command and added a note instead. :) Thanks

File: content/v2.0/get-started.md [hunk 1]
```diff
@@ -171,6 +171,8 @@ the [InfluxDB HTTP API](/v2.0/reference/api/).
 docker run --name influxdb -p 9999:9999 quay.io/influxdb/influxdb:2.0.0-beta
 ```
 
+Note: To run influxdb in detached mode use flag -d in the above command.
```
Comment (sanderson) [inline content/v2.0/get-started.md]: Quick edit:

```
_To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._
```

Comment (sanderson) [reply content/v2.0/get-started.md]: @VijetaH Once you apply this edit ☝️, we'll get this merged and deployed.

Comment (VijetaH) [reply content/v2.0/get-started.md]: @sanderson oops i resolved before applying edit. Its updated now.

**Pull request merged**